core/src/avm2/globals/flash/desktop/Clipboard.as

```diff
@@ -0,0 +1,54 @@
+package flash.desktop {
+    import __ruffle__.stub_getter;
+    import __ruffle__.stub_method;
+    import flash.system.System;
+
+    public class Clipboard {
+        private static var _generalClipboard = new Clipboard();
+
+        public static function get generalClipboard(): Clipboard {
+            return Clipboard._generalClipboard;
+        }
+
+        function Clipboard() {
+            // TODO: This should only be callable in AIR
```

Comment:
Why not add the `AIR` annotation here?

Reply:
Also, I guess this is another case of the docs lying then?
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/desktop/Clipboard.html#Clipboard()

Reply:
The constructor exists in Flash, you just get an error in Flash when calling:
```
Error: Error #2178: The Clipboard.generalClipboard object must be used instead of creating a new Clipboard.
```

I didn't try to figure out how to implement this, because I needed a constructor for `generalClipboard`.

Reply:
Okay, fair enough.

Reply:
If there's a way to get whether the current runtime is AIR runtime then it can be implemented by:
if(!isAirRuntime)
  if(_generalClipboard)
    throw new Error("Error #2178: The Clipboard.generalClipboard object must be used instead of creating a new Clipboard.");

Reply:
Cool idea.